mobilizon_reshare/storage/db.py

```diff
@@ -17,7 +19,18 @@ def __init__(self, path: Path):
         if not self.is_init:
             self.path.parent.mkdir(parents=True, exist_ok=True)
 
+    async def __implement_db_changes__(self):
```

Comment:
in python questo formato per i nomi funzione è riservato ai magic methods. Un utente non dovrebbe mai usarlo. Se vuoi fare un metodo privato è sufficiente usare un solo underscore prima del nome

Reply:
fatto